### PR TITLE
SELU & Derivative

### PR DESCRIPTION
**Pull request by raver119**
SELU & SELUDerivative ops

As requested here: https://github.com/deeplearning4j/nd4j/issues/1878

### CODE REVIEW

File: include/ops/ops.h [hunk 1]
```diff
@@ -1063,6 +1066,28 @@ namespace simdOps {
 		}
 	};
 
+    template<typename T>
+    class SELU {
```
Comment (maxpumperla) [inline include/ops/ops.h]: Overall, if I see the context of this addition, the indentation seems off by a tab or so.

Comment (maxpumperla) [inline include/ops/ops.h]: Otherwise, straight forward enough, LGTM!

**Comment by raver119** [comment]
screenshot from 2017-06-16 00-21-12

indentation looks as intended in CLion

**Comment by maxpumperla** [comment]
huh, the github preview must play tricks on me, looks indeed fine. sorry.

File: include/ops/ops.h [hunk 2]
```diff
@@ -1063,6 +1066,28 @@ namespace simdOps {
...
+    };
+
+    template<typename T>
+    class SELUDerivative {
+    public:
+        no_op_exec_special
+        no_op_exec_special_cuda
+
+        op_def static T op(T d1, T *params) {
+            return d1 > (T) 0.0f ? (T) SELU_LAMBDA : (T) SELU_LAMBDA * nd4j::math::nd4j_exp<T>(d1);
```
Comment (AlexDBlack) [inline include/ops/ops.h]: ```d/dx( L * (a * e^x - a) ) = L * a * e^x```
vs. (incorrect) ```L * e^x``` here

**Comment by raver119** [comment]
Thanks, fixed!

**Pull request merged**